### PR TITLE
chore: migrate to GitHub actions

### PR DESCRIPTION
**Pull request by nikku**
This migrates to GitHUb ACtions

### CODE REVIEW

**Comment by nikku** [comment]
Can be squashed before merge.

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        os: [ ubuntu-latest ]
+        node-version: [ 14 ]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v1
```
Comment (MaxTru) [inline .github/workflows/CI.yml:17]: Any reason we use v1 and not v2 considering we are now starting with GitHub Actions (which is the latest) (Sorry if we already decided something here, I know that @nikku and @pinussilvestrus discussed it already, but I dont know the outcome)

**Review by MaxTru — APPROVED** [review]
Approved, can be merged.

My comment (setup-node@v1 vs setup-node@v2) can be addressed within the overall context of the GitHub Actions migration.

Nice who "smoothly" this migration works. I learnt from this!

File: .github/workflows/CI.yml (hunk 1 above)
Comment (nikku) [reply .github/workflows/CI.yml:17]: No particular reason. Happy to take your PR on our default workflow templates that up the version there: 
https://github.com/bpmn-io/.github/tree/master/workflow-templates.

**Pull request merged**